netharn/models/yolo2/light_yolo.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, stride=2):
         self.darknet = True
 
     def __repr__(self):
-        return f'{self.__class__.__name__} (stride={self.stride}, darknet_compatible_mode={self.darknet})'
+        return '{self.__class__.__name__} (stride={self.stride}, darknet_compatible_mode={self.darknet})'
```

Comment:
I'm going to merge this because using format strings breaks 3.5 support, but a future commit should fix this to format the string with the proper information.

Reply:
e.g. : `'{} (stride={}, darknet_compatible_mode={})'.format(self.__class__.__name__, self.stride, self.darknet)`